src/getItems.js

```diff
@@ -1,9 +1,35 @@
+/* eslint-disable max-len */
 const Apify = require('apify');
 
 const { log } = Apify.utils;
+const { LABEL } = require('./consts');
 
-async function getItems(pageObj, pageData, resultsArr) {
-    // Scrape all items that match the selector
+async function getItems(pageObj, pageData, resultsArr, label) {
+    if (label === LABEL.NEW) {
+        // need to wait after page 1
+        await pageObj.waitForSelector('.zg-grid-general-faceout');
+
+        for (let i = 1; i < 5; i++) {
+            await Apify.utils.puppeteer.infiniteScroll(pageObj, { scrollDownAndUp: true, timeoutSecs: i, waitForSecs: 5 });
```

Comment:
this is mostly due `headless: true`, and/or the tab staying at the background instead of the foreground. you can force it to the foreground by using `await page.bringToFront()`, but only works when 1 instance contains just 1 tab.

Reply:
Tried to play around a bit but getting inconsistent results still. I'll try more a bit

Also thanks for the review!